platform/windows/display_server_windows.cpp

```diff
@@ -328,6 +328,15 @@ Size2i DisplayServerWindows::screen_get_size(int p_screen) const {
 	return data.size;
 }
 
+float DisplayServerWindows::screen_get_refresh_rate(int p_screen) const {
+	_THREAD_SAFE_METHOD_
+
+	DEVMODE data;
+	memset(&data, 0, sizeof(data));
+	EnumDisplaySettingsA(NULL, ENUM_CURRENT_SETTINGS, &data);
+	return data.dmDisplayFrequency;
+}
+
```

Comment:
I guess it should get the current display refresh rate, not the main one. Also, setting `xxSize` field value is usually required for the most Windows API functions.

Something like (not tested):

```suggestion
typedef struct {
	int count;
	int screen;
	float rate;
} EnumRefData;

static BOOL CALLBACK _MonitorEnumProcRef(HMONITOR hMonitor, HDC hdcMonitor, LPRECT lprcMonitor, LPARAM dwData) {
	EnumRefData *data = (EnumRefData *)dwData;
	if (data->count == data->screen) {

		MONITORINFOEXW mi;
		memset(&mi, 0, sizeof(mi);
		mi.cbSize = sizeof(mi);
		GetMonitorInfoW(hMonitor, &mi);

		DEVMODE dm;
		memset(&dm, 0, sizeof(dm));
		dm.dmSize = sizeof(dm);
		EnumDisplaySettingsW(mi.szDevice, ENUM_CURRENT_SETTINGS, &dm);
		
		data->rate = dm.dmDisplayFrequency;
	}

	data->count++;
	return TRUE;
}

float DisplayServerWindows::screen_get_refresh_rate(int p_screen) const {
	_THREAD_SAFE_METHOD_

	EnumRefData data = { 0, p_screen == SCREEN_OF_MAIN_WINDOW ? window_get_current_screen() : p_screen, 0 };
	EnumDisplayMonitors(nullptr, nullptr, _MonitorEnumProcRef, (LPARAM)&data);
	return data.rate;
}
```